expo/scheduler/models.py

```diff
@@ -6,7 +6,10 @@
 from model_utils.managers import InheritanceManager
 from gbetext import *
 from gbe.expomodelfields import DurationField
-from scheduler.functions import set_time_format
+from expo.settings import (
+    DATETIME_FORMAT,
+    DAY_FORMAT,)
```

Comment:
No, set_time_format was setting it to %a, %H:%M %p, which is not DATE_FORMAT, having hours and minutes.  DATETIME_FORMAT is closest.  And this is the model layer, so if we want to change it, it should change to a UNIX time object instead of a formatted date/time string.  Too bad that would break other things right now....